propositions_and_proofs.md

```diff
@@ -43,10 +43,10 @@ constant of such a type.
 #   proof : p
 #check Proof   -- Proof : Prop → Type
 
-axiom and_comm (p q : Prop) : Proof (Implies (And p q) (And q p))
+axiom and_commutative (p q : Prop) : Proof (Implies (And p q) (And q p))
```

Comment:
We now have `and_comm` in scope by default. I chose to rename it here, rather than secretly stick it in a namespace, because I want code bits to work if people type them in.

On the other hand, the hidden `Proof` structure is not present, so this is a very weak argument for this solution. Happy to do the other one if you'd prefer.